package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "trusona-server-sdk",
-  "version": "1.4.2",
+  "version": "2.0.0",
```

Comment:
Will you say more about why this is a major version upgrade?

Reply:
Removing AP region support could potentially break existing projects that use it.